pkg/signal/signal_linux.go

```diff
@@ -1,8 +1,14 @@
-// +build !mips,!mipsle,!mips64,!mips64le
+// +build linux
```

Comment:
should we add in a copyright line at the top, even if it wasn't present in the old file?

Reply:
Absolutely. Thanks for noticing!

Reply:
Done :heavy_check_mark: